azurelinuxagent/ga/update.py

```diff
@@ -251,6 +251,15 @@ def run(self, debug=False):
             logger.info(u"Agent {0} is running as the goal state agent",
                         CURRENT_AGENT)
 
+            # Log OS-specific info, locally and as a telemetry event.
+            msg = u"Distro info: {0}:{1}, osutil class being used: {2}, " \
```

Comment:
Not sure osutil should be logged, but if it should probably it'd be better to do it in get_osutil

Reply:
I would also recommend adding other info - example:

```
osversion = "{0}:{1}-{2}-{3}:{4}".format(platform.system(),
                                                 DISTRO_NAME,
                                                 DISTRO_VERSION,
                                                 DISTRO_CODE_NAME,
                                                 platform.release())
```

Reply:
`get_osutil` is called all over the place, I don't think we want to emit an event every time it's called.

The purpose of this event was to confirm we were getting the correct data regarding distro and assert the changes made in #1580 were correct (hence the `osutil` parameter).

@vrdmr is correct, we are already logging distro-specific info so it might be better to make this a release-specific event then? Although I think it would be very useful to easily query Kusto for `agent service name` and see which distro uses which. It's something that required manual testing in the past.